lib/scsi_defs.h

```diff
@@ -73,12 +76,19 @@ namespace scsi_defs {
   };
 
   // SCSI Reference Manual Table 10 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
-  typedef struct ControlByte {
+  struct ControlByte {
     uint8_t obsolete : 2;
     bool naca : 1;
     uint8_t reserved : 3;
     uint8_t vendorSpecific : 2;
```

Comment:
Can this PR also change this variable to snake_case? This would close all style issues in master

Reply:
done!